static/lang/en.json

```diff
@@ -3805,6 +3805,7 @@
         "TrainedSkillsLabel": "Trained Skills",
         "TraitAberration": "Aberration",
         "TraitAcid": "Acid",
+        "TraitAdditive": "Additive",
```

Comment:
If you're adding this then you're gonna want to actually add the trait too

Reply:
Where do I add it?

Reply:
https://github.com/foundryvtt/pf2e/blob/6f190d941f17d03739a2e0e37e235b8241437795/src/scripts/config/traits.ts